smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -174,29 +137,18 @@ contract MixinTransfer is
   }
 
   /**
-   * @notice An ERC-20 style transfer.
-   * @param _value sends a token with _value * expirationDuration (the amount of time remaining on a standard purchase).
-   * @dev The typical use case would be to call this with _value 1, which is on par with calling `transferFrom`. If the user
-   * has more than `expirationDuration` time remaining this may use the `shareKey` function to send some but not all of the token.
+   * @notice An ERC-721 style transfer.
```

Comment:
That's actually an ERC20 style transfer :) as ERC721 does not have transfers.

```suggestion
   * @notice An ERC-20 style transfer.
```

Reply:
We added it back in the day to make it easy to transfer from MetaMask which has ERC20 transfers.